README.md

```diff
@@ -142,6 +142,29 @@ steps:
       MERGE_METHOD: MERGE
 ```
 
+### Presets
+
+Presets enable additional functionality which can be used to better personalize
+default behavior of the Merge me! Action.
+
+Available presets are:
+
+- `DEPENDABOT_MINOR` - Merge only minor and patch dependency updates for pull
+  requests created by Dependabot if the dependency version follows
+  [Semantic Versioning v2](https://semver.org/).
+- `DEPENDABOT_PATH` - Merge only patch dependency updates for pull requests
```

Comment:
Sorry a little late to the party, but surely here it was supposed to be `PATCH`

Reply:
Thanks for pointing that out @jraoult. I opened a PR for it #545